src/apps/test/pfutest.cpp

```diff
@@ -0,0 +1,133 @@
+#include "pfu.hpp"
+#include "../test/MemFile.hpp"
```

Comment:
I'd suggest adding a `target_include_directory` CMake thingy or similar to pick these up. fine as is though, I just "like" not having `../` type relative things in when there's other solutions.